srcpkgs/xournalpp/INSTALL.msg

```diff
@@ -1,4 +1,5 @@
-  You  need to install the standalone package to  use LaTeX with
-  xournalpp. The following command will install it (needs root privileges).
-
+You  need install LaTeX and the standalone package in order to take
+advantage of some xournalpp features. You can install them either by
+installing texlive-latexextra or by installing texlive-bin and then
+using tlmgr (needs root privileges):
   'tlmgr install standalone'
```

Comment:
Please drop `INSTALL.msg` entirely. `xournalpp` provides meaningful errors about needing `pdflatex` when you try to use a feature that requires it, so we don't need extra noise here.

Reply:
Done ;)